docs/internals/contributing/committing-code.txt

```diff
@@ -74,7 +73,7 @@ line.
 When rewriting the commit history of a pull request, the goal is to make
 Django's commit history as usable as possible:
 
-* If a patch contains back-and-forth commits, then rewrite those into one.
+* If a PR contains back-and-forth commits, then rewrite those into one.
```

Comment:
Can use "contribution" here instead of PR

Reply:
Agreed, changed 👍🏻